README.md

```diff
@@ -1,38 +1,64 @@
 # Slice for Visual Studio Code
 
-This language extension adds support for highlighting Slice files within Visual Studio Code.
-It supports highlighting the following file types:
+This extension provides support for the Slice Interface Definition Language (IDL).
+
+## Features
+
+- Syntax highlighting
+- Syntax validation
+- Error detection & reporting
+- 'Go to Definition' jumping
```

Comment:
I would just write:
```
- Go to Definition
```

No jumping, no quotes.

Reply:
I think it would be clearer to explain in this top section that most features only apply to .slice files.

Reply:
Fixed.

Reply:
Did you push?